source/how-we-work/how-to-run-a-design-system-team-ceremony-or-workshop.html.md

```diff
@@ -0,0 +1,165 @@
+
+It can sometimes be difficult to articulate the goal if you do not have all the context. Ask the team to edit the goal either during the session or afterwards. There are prompts to follow within the GitHub issue template to achieve a well articulated goal.
+
+It’s important to ensure each goal has an assignee(s) who will work on the goal. Do this during the session.
+
+At the end of the session, ensure all goals are in the “Sprint backlog” ready to be worked on and assigned to the correct “Iteration” on the board (eg Q2 22/23 Sprint 2).
+
+### Backlog gardening
+
+Backlog gardening occurs for the Frontend squad within the team every other Tuesday.
+
+The purpose of this session is to make sure that the backlog has all the right work in it, that the work is well described, and that it's ordered well. It’s a fancy way of saying backlog refinement.
+
+Think ahead over the next 4 weeks and consider which epics – or tasks in epics – the team is likely to start working on soon. Ensure that the outcomes are clear and any dependencies are noted: the goal is to make sure that an issue is well described and that someone could pick it up independently. For example, if it’s a spike that should result in a team workshop or learning session, make sure that’s a part of the issue or epic.
+
+This is also an opportunity to think about any pre-work or knowledge sharing that needs to happen. For example, if a piece of delivery work needs scoping, you’ll probably want to arrange a team workshop to help, like MoSCoW prioritisation or getting feedback from service teams.
+
+Where a piece of work can help someone meet their professional development objectives, highlight that they might want to work on it – or assign it to them and let others on the team know they’ll be tackling it.
+
+Note: This ceremony is fairly organic and has suited the team OK so far, but we have identified a need to increase collaboration and pairing on the team. So it should probably be amended / adapted as we tweak our processes to meet the needs of the team.
+
+### Mid-sprint review
+
+A mid-sprint review is a checkpoint halfway through a sprint. The session usually lasts 30 mins and checks progress against sprint goals, providing the opportunity to pivot if needed..
+
+In a mid-sprint review, the squad asks the following questions of the sprint goals:
+- have any risks or dependencies emerged?
+- how can blockers be resolved?
+- has the scope changed?
+- has it been deprioritised? If so, why?
+- has a new goal emerged?
+
+Update the sprint board as a result of the discussion.
+
+### Retrospectives
+
+Retros occur every other Tuesday for each squad. The purpose of retro is to reflect on the previous two weeks and identify ways the team can improve going forward. Therefore it’s useful if the whole team attends so we collectively agree to actions.
```

Comment:
```suggestion
Retros occur every other Tuesday for each squad. The purpose of retro is to reflect on the previous two weeks and identify ways the team can improve its processes going forward. Therefore it’s useful if the whole team attends so we collectively agree to actions.
```

Reply:
This suggestion might be doing a lot of heavy-lifting, but the intent is to highlight how retros are more about our methods and processes. Feel free to remove it or reword it though!